Cargo.lock

```diff
@@ -2747,7 +2747,7 @@ version = "0.0.0"
 dependencies = [
  "graphviz 0.0.0",
  "log 0.4.6 (registry+https://github.com/rust-lang/crates.io-index)",
- "rand 0.5.5 (registry+https://github.com/rust-lang/crates.io-index)",
+ "rand 0.6.1 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Any chance of going to 0.6.5 instead? IIUC, that should bring in rand_os 0.1.3, which fixes SIGSYS crashes on illumos.

Reply:
I cannot decide that on my own, what do you think @scottmcm? (I updated from 0.6.1 to 0.6.5 on local, but the build was failed.)

Reply:
@JohnTitor do you have error message from that build?

Reply:
@mati865 Yes, the message is here:
```
error[E0463]: can't find crate for `core`=============>                ] 16/22: libc
   --> C:\Users\username\.cargo\registry\src\github.com-1ecc6299db9ec823\libc-0.2.46\src\lib.rs:161:1
    |
161 | extern crate std as core;
    | ^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error: Could not compile `libc`.
```
I updated rand 0.6.1 by `cargo update -p rand:0.6.1` and changed [this line](https://github.com/rust-lang/rust/blob/7c19e1eed5418b8c02be65d678417b241ee01a3e/src/libstd/Cargo.toml#L29) to 0.6.

Reply:
Looks like `std` failed to build, either it's issue with cache/incremental compilation or something really bad happened.